src/core/Form/Toggle/Toggle.baseStyles.tsx

```diff
@@ -52,10 +53,10 @@ export const baseStyles = ({ theme = suomifiTheme }: ToggleProps) => css`
         transform: translateX(50%);
       }
       .${svgPrefix}fi-toggle-icon-slide {
-        fill: ${theme.colors.toggleOnSlide};
+        fill: ${alphaHex(0.5)(theme.colors.successBase)};
```

Comment:
should we have this color without need for opacity change?

Reply:
That would be PAMU's decision